docs/source/markdown/podman-create.1.md

```diff
@@ -750,6 +750,9 @@ Security Options
 - `seccomp=unconfined` : Turn off seccomp confinement for the container
 - `seccomp=profile.json` :  White listed syscalls seccomp Json file to be used as a seccomp filter
 
+- `procfs-opts=OPTIONS` : Comma separated list of options to use for the /proc mount.
+
```

Comment:
Can you mention the security options here?

Reply:
these are kernel specific.  Is it fine to add something like: `More details for the possible mount options are specified at **proc(5)** man page.` ?  So we don't have to worry to keep it updated